recipes/simwalk/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: simwalk2
+  version: "2.91"
+
+source:
+  fn: SimWalk2-2.91-lin.tar.gz # [linux]
+  url: https://github.com/BioTools-Tek/packages/blob/master/simwalk2/SimWalk2-2.91-lin.tar.gz?raw=true # [linux]
+  md5: 2a529d50ca48cd7ea2b67f85c290f203 # [linux]
+
+  fn: SimWalk2-2.91-osx.tar.gz # [osx]
+  url: https://github.com/BioTools-Tek/packages/blob/master/simwalk2/SimWalk2-2.91-osx.tar.gz?raw=true # [osx]
+  md5: 157cf4a869afdacf8e415005886f27c7 # [osx]
+
+  
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - libgfortran # [linux]
+    - libgcc # [linux]                                                                                                                                    
+  run:                                                                                                                                                
+    - libgfortran # [linux]
+    - libgcc # [linux]
+    - zlib # [linux]
+
+test:
+  source_files:
```

Comment:
This will not work you don't have files included here.
Can you just call `simwalk2 --help` or similar and catch maybe the expected error.

Reply:
simwalk unfortunately takes no arguments, it just looks for a `BATCH2.DAT` file which specifies where all the other input files are.

I will copy over the relevant files from the work directory, thanks